### PR TITLE
enabeld is not required anymore and code formatting

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Admin/CategoryAdmin.php [hunk 1]
```diff
@@ -98,10 +100,16 @@ protected function configureFormFields(FormMapper $formMapper)
...
         $position = $this->hasSubject() && !is_null($this->getSubject()->getPosition()) ? $this->getSubject()->getPosition() : 0;
 
-        $formMapper->end()
+        $formMapper
+            ->end()
             ->with('Options', array('class' => 'col-md-6'))
-                ->add('enabled')
-                ->add('position', 'integer', array('required' => false, 'data' => $position))
+                ->add('enabled', null, array(
+                    'required' => false,
```
Comment (OskarStark) [inline Admin/CategoryAdmin.php]: this is the code change, otherwise it has to be enabled everytime

**Comment by Soullivaneuh** [comment]
Can you please explain why this is not required anymore?

**Comment by OskarStark** [comment]
sure

bildschirmfoto 2015-10-07 um 10 48 55

when i have to check "Aktiviert?" (which is enabled) all the time, i can't deactivate it...

**Comment by Soullivaneuh** [comment]
Ok, please rebase.

**Comment by OskarStark** [comment]
rebase done @Soullivaneuh

**Comment by Soullivaneuh** [comment]
CS need to be fixed. :-)

**Pull request merged**